testing/termite/APKBUILD

```diff
@@ -0,0 +1,66 @@
+# Contributor: Ian Bashford <ianbashford@gmail.com>
+# Maintainer: Ian Bashford <ianbashford@gmail.com>
+pkgname=termite
+pkgver=15
+pkgrel=0
+_vtever=0.50.2-ng
+pkgdesc="A simple VTE-based terminal"
+url="https://github.com/thestinger/termite/"
+arch="x86_64"
+options="!check"  # No test suite.
+license="LGPL"
```

Comment:
This is not an SPDX compliant license identifier

Reply:
Done